src/Neo/UInt160.cs

```diff
@@ -123,17 +128,22 @@ public void Serialize(Span<byte> destination)
             }
             else
             {
-                const int IxValue2 = sizeof(ulong);
-                const int IxValue3 = sizeof(ulong) * 2;
-
-                Span<byte> buffer = stackalloc byte[Length];
-                BinaryPrimitives.WriteUInt64LittleEndian(buffer, _value1);
-                BinaryPrimitives.WriteUInt64LittleEndian(buffer[IxValue2..], _value2);
-                BinaryPrimitives.WriteUInt32LittleEndian(buffer[IxValue3..], _value3);
-                buffer.CopyTo(destination);
+                SerializeBigEndian(destination);
             }
         }
 
+        internal void GetSpanLittleEndian(Span<byte> destination)
```

Comment:
```suggestion
        internal void SafeSerialize(Span<byte> destination)
```
?? the other method it's unsafe (faster)